paranthetics.py

```diff
@@ -0,0 +1,35 @@
+"""Check a string to see if all the parantheses are closed."""
+
+
+from que import Queue
+
+
+def check_parens(str):
```

Comment:
Watch the variable name! Inside the scope of this function, you would no longer have access to the built-in behavior for the `str` type object.